specification/maps/data-plane/Route/preview/2023-10-01-preview/route.json

```diff
@@ -766,7 +766,7 @@
               "readOnly": true
             },
             "durationInSeconds": {
-              "description": "The time that it takes, in seconds, to travel a corresponding TravelDistance.",
+              "description": "Estimated travel time in seconds property without traffic conditions. This value is provided if optimizeRoute does not include traffic considerations.",
```

Comment:
@koyasu221b One more place we missed, or did we not update because this field returns a different ETA than other duration field?